### ISSUE TITLE
LDAP Mapping in GUI not saving

### ISSUE THREAD

**Issue by krispayne** [ansible/awx#956]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - UI

##### SUMMARY
I have 44 lines of JSON for my LDAP Team Map, when I go to another area of the interface and come back, there is only the first 7 lines.

##### ENVIRONMENT
* AWX version: 1.0.2.307
* AWX install method: docker on linux
* Ansible version:  2.4.2.0
* Operating System: CentOS 7
* Web Browser: Chrome latest

##### STEPS TO REPRODUCE

I am setting up my LDAP Team mappings and I have 6 Teams that I am attempting to map to 6 LDAP groups. When I input the JSON and save it, it remains and the mappings work. When I then go to another page and come back, the JSON is only the first mapping, and not the latest JSON that I input.

Here is what I input and save:
```
{
"Software": {
  "organization": "company",
  "users": "cn=sweng,cn=groups,cn=accounts,dc=company,dc=net",
  "remove": true,
  "remove_admins": true
},

"Database": {
	"organization": "company",
	"users": "cn=dba,cn=groups,cn=accounts,dc=company,dc=net",
	"remove": true,
	"remove_admins": true
},

"Linux Build": {
	"organization": "company",
	"users": "cn=it,cn=groups,cn=accounts,dc=company,dc=net",
	"remove": true,
	"remove_admins": true
},

"Security": {
	"organization": "company",
	"users": "cn=it,cn=groups,cn=accounts,dc=company,dc=net",
	"remove": true,
	"remove_admins": true
},

"Storage": {
	"organization": "company",
	"users": "cn=it,cn=groups,cn=accounts,dc=company,dc=net",
	"remove": true,
	"remove_admins": true
},

"Systems": {
	"organization": "company",
	"users": "cn=it,cn=groups,cn=accounts,dc=company,dc=net",
	"remove": true,
	"remove_admins": true
}

}
```
This is what's shown in the box after I navigate around a bit:
```
{
 "Software": {
  "organization": "company",
  "users": "cn=sweng,cn=groups,cn=accounts,dc=company,dc=net",
  "remove": true
 }
}
```

##### EXPECTED RESULTS

When I save the JSON in the LDAP mapping, it should save and be there.

##### ACTUAL RESULTS

And old version of what I saved is presented.

**Comment by krispayne** [comment]
This is also happening in the Organization mapping.

**Comment by krispayne** [comment]
I think I figured this out.

If I go to /api/v2/settings/ldap and input it myself, it throws an error about "remove_admins": true not being valid. I removed those entires and clicked patch. It shows my mappings in the GUI now. The GUI never showed this error.

**Comment by alancivita** [comment]
is there any progress on this bug..on the latest version is still there and ldap credentials still cannot be saved

**Comment by fia5000** [comment]
I can replicate this bug. None of the ldap config gets saved

**Comment by mhurron** [comment]
Seeing same results on AWX docker container 1.0.6.3.

Docker Log:
2018-05-02 17:06:54,357 WARNING  awx.api.generics status 400 received by user admin attempting to access /api/v2/settings/all/ from 172.20.0.7
[pid: 29|app: 0|req: 187/2256] 172.20.0.7 () {54 vars in 4588 bytes} [Wed May  2 17:06:53 2018] PATCH /api/v2/settings/all/ => generated 143 bytes in 394 msecs (HTTP/1.1 400) 8 headers in 252 bytes (1 switches on core 0)
172.20.0.7 - - [02/May/2018:17:06:54 +0000] "PATCH /api/v2/settings/all/ HTTP/1.1" 400 143 "https://awx.example.com/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_4) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/11.1 Safari/605.1.15"

Browser Javascript error log:

[Error] Failed to load resource: the server responded with a status of 400 (Bad Request) (all, line 0)
[Error] undefined is not an object (evaluating 'data[field]')
https://awx.example.com/static/js/app.6489c481c2a5ca9d0668.js:42 – "Possibly unhandled rejection: {\"line\":42,\"column\":863385,\"sourceURL\":\"https://awx.example.com/static/js/app.6489c481c2a5ca9d0668.j…"
	(anonymous function) (vendor.4f5ff2b0e1c182be9654.js:583:93671)
	processChecks (vendor.4f5ff2b0e1c182be9654.js:583:135227)
	$digest (vendor.4f5ff2b0e1c182be9654.js:583:146524)
	$apply (vendor.4f5ff2b0e1c182be9654.js:583:149482)
	done (vendor.4f5ff2b0e1c182be9654.js:583:101285)
	completeRequest (vendor.4f5ff2b0e1c182be9654.js:583:106305)
	onload (vendor.4f5ff2b0e1c182be9654.js:583:107091)

EDIT: If I follow the instructions at https://github.com/ansible/awx/issues/1704#issuecomment-378142089 and fill in USER DN Template it spits out the same traceback as #1704

**Comment by ryanpetrello** [comment]
We've just released 1.0.7, which we believe resolves the underlying issue here. You can try it out here: https://github.com/ansible/awx/releases/tag/1.0.7

Let us know if you're still seeing this issue after installing the latest awx - thanks!
